dotnet/package/microsoft.workloads.csproj

```
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
-    <PackageId>Microsoft.NET.Workload.$(_PlatformName)</PackageId>
+    <PackageId>Microsoft.NET.Sdk.$(_PlatformName).Manifest-6.0.100</PackageId>
```

You could do `export DOTNET6_VERSION_BAND` somewhere in `dotnet/Makefile`, and then you can use that variable here:

```suggestion
    <PackageId>Microsoft.NET.Sdk.$(_PlatformName).Manifest-$(DOTNET6_VERSION_BAND)</PackageId>
```

---

I just passed it with `-p:VersionBand`, this works, thanks.